### PR TITLE
Added CustomGlobalExceptionHandler class, Added validation for existi…

### PR DESCRIPTION
**Pull request by AgroFix**
…ng DTO classes

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @Size(max = 255, message = "Text limit 255 symbols")
```
Comment (nshtykh) [inline src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java:13]: This message is repeated, you can make it a constant

**Review by nshtykh — APPROVED** [review]
Great job!

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java (hunk 1 above)
Comment (bhdnchui) [reply src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java:13]: It is not critical for messages, but it is good point =)

**Review by bhdnchui — APPROVED** [review]
Good job!

**Pull request merged**